examples/speech_to_speech2/eval.sh

```diff
@@ -0,0 +1,6 @@
+simuleval \
```

Comment:
Thanks for adding this Mandy! Rather than adding a shell script, could you move this to the `tests` directory and add a python unit test following the pattern of `https://github.com/facebookresearch/SimulEval/blob/main/simuleval/test/test_agent.py`?

Reply:
Hi Anna! Thanks for the feedback. I just committed some changes to my branch. I was just wondering if there's supposed to be a bit of code repetition between the agent english_alternate_agent.py under speech_to_speech and the policy function under test_s2s.py?

Reply:
I think it's ok if there's a bit of repetition; this is intended as more of an illustrative example and doesn't need to be refactored too much